content/webapp/components/IIIFViewer/ViewerSidebar.tsx

```diff
@@ -160,8 +184,42 @@ const ViewerSidebar: FunctionComponent<ViewerSidebarProps> = ({
     note => note.noteType.id === 'location-of-original'
   );
 
+  const digitalLocationInfo =
+    digitalLocation && getDigitalLocationInfo(digitalLocation);
+
+  const isWorkVisibleWithPermission =
+    digitalLocationInfo?.accessCondition === 'restricted' &&
```

Comment:
Using the digitalLocation accessCondition isn't going to work for things that have a mix of restricted and non restricted things, which is ok if we handle that case differently. If not we'll need to use the [auth object on the TransformedManifest](https://github.com/wellcomecollection/wellcomecollection.org/blob/4965b6e75cd29e89c3d5fdec8f3665d6ddde414a/content/webapp/types/manifest.ts#L100) to determine the presence of restricted things. If there is an auth.v1. externalAccessService value (or a auth.v2. externalAccessService value if the toggle is on) then that indicates something is restricted.

Reply:
OK so I had a look and I think because we don't have the copy/behaviour for partly restricted yet, might be better off not addressing them at all _for now_. Then when we work on #11278 we amend the logic if needed.

I just know the message might not be showing in the same spot at all, so I'm thinking it's better to wait?

If you're happy to, I'll be linking this thread in that ticket because it's really useful.